src/library/colordelegate.cpp

```diff
@@ -13,17 +13,17 @@ ColorDelegate::ColorDelegate(QTableView* pTableView)
 }
 
 void ColorDelegate::paintItem(QPainter* painter, const QStyleOptionViewItem& option, const QModelIndex& index) const {
-    DEBUG_ASSERT(index.data().isValid());
-    if (index.data().isNull()) {
+    const auto color = mixxx::RgbColor::optional(index.data());
+
+    if (!color.has_value()) {
```

Comment:
has_value() is not supported on Xenial, only operator bool()

```
if (!color) {
    ...
}
```

Reply:
The code has become both more readable and maintainable by this simple reordering :+1:

Reply:
Done.